src/config.rs

```diff
@@ -56,12 +60,17 @@ pub struct Options {
     #[serde(skip_deserializing, default)]
     pub disable_hygiene: bool,
 
+    #[cfg(target_arch = "wasm32")]
```

Comment:
Why did you do this?

Reply:
The real problem is line 59 that removes the property:

    #[cfg(not(target_arch = "wasm32"))]
    #[serde(skip_deserializing, default)]
    pub disable_hygiene: bool

it breaks line 250:
         .hygiene(!self.disable_hygiene)

My fix only added the property back when compiling for wasm

Reply:
I checked it!